qualifier.py

```diff
@@ -0,0 +1,80 @@
+"""
+Use this file to write your solution for the Summer Code Jam 2020 Qualifier.
+
+Important notes for submission:
+
+- Do not change the names of the two classes included below. The test suite we
+  will use to test your submission relies on existence these two classes.
+
+- You can leave the `ArticleField` class as-is if you do not wish to tackle the
+  advanced requirements.
+
+- Do not include "debug"-code in your submission. This means that you should
+  remove all debug prints and other debug statements before you submit your
+  solution.
+"""
+import datetime
+import typing
+import re
+
+
+class ArticleField:
+    """The `ArticleField` class for the Advanced Requirements."""
+
+    def __init__(self, field_type: typing.Type[typing.Any]):
+        pass
+
+
+class Article:
+    """The `Article` class you need to write for the qualifier."""
+
+    def __init__(
+        self, title: str, author: str, publication_date: datetime.datetime, content: str
+    ):
+        self.title = title
+        self.author = author
+        self.publication_date = publication_date
+        self.content = content
+
+    def __repr__(self):
+        return f'<Article title="{self.title}" author=\'{self.author}\' publication_date=\'{self.publication_date.isoformat("T")}\'>'
```

Comment:
Strings should not be split over multiple lines without any escape at all. It is recommend to escape in parantheses (note that there is no `,` in-between the strings, it is **not** a `tuple`).

```py
return (
    f'<Article title="{self.title}" author=\'{self.author}\' '
    f'publication_date=\'{self.publication_date.isoformat("T"}\'>'
)
```